www/assets/locales/locale-es.json

```
@@ -2,52 +2,52 @@
   "nutriments": {
     "kilojoules": "Kilojulios",
     "calories": "Calorías",
-    "fat": "Fat",
-    "saturated-fat": "Sat fat",
-    "carbohydrates": "Carbs",
-    "sugars": "Sugars",
-    "fiber": "Fiber",
-    "proteins": "Proteins",
-    "salt": "Salt",
-    "sodium": "Sodium",
-    "cholesterol": "Cholesterol",
-    "trans-fat": "Trans fat",
-    "monounsaturated-fat": "Monounsat fat",
-    "polyunsaturated-fat": "Polyunsat fat",
-    "omega-3-fat": "Omega-3 fat",
-    "omega-6-fat": "Omega-6 fat",
-    "omega-9-fat": "Omega-9 fat",
-    "vitamin-a": "Vitamin A",
-    "vitamin-b1": "Vitamin B1 (Thiamine)",
-    "vitamin-b2": "Vitamin B2 (Riboflavin)",
-    "vitamin-pp": "Vitamin B3 (Niacin)",
-    "pantothenic-acid": "Vitamin B5 (Pantothenic acid)",
-    "vitamin-b6": "Vitamin B6 (Pyridoxine)",
-    "biotin": "Vitamin B7 (Biotin)",
-    "vitamin-b9": "Vitamin B9 (Folate)",
-    "vitamin-b12": "Vitamin B12 (Cobalamin)",
-    "vitamin-c": "Vitamin C",
-    "vitamin-d": "Vitamin D",
-    "vitamin-e": "Vitamin E",
-    "vitamin-k": "Vitamin K",
-    "potassium": "Potassium",
-    "chloride": "Chloride",
-    "calcium": "Calcium",
-    "phosphorus": "Phosphorus",
-    "iron": "Iron",
-    "magnesium": "Magnesium",
+    "fat": "Grasa",
+    "saturated-fat": "Grasa saturada",
+    "carbohydrates": "Carbohidratos",
+    "sugars": "Azúcares",
+    "fiber": "Fibra",
+    "proteins": "Proteínas",
+    "salt": "Sal",
+    "sodium": "Sodio",
+    "cholesterol": "Colesterol",
+    "trans-fat": "Grasas trans",
+    "monounsaturated-fat": "Grasa Monoinsaturada",
+    "polyunsaturated-fat": "Grasa Polyinsaturada",
+    "omega-3-fat": "Acidos Grasos Omega-3",
+    "omega-6-fat": "Acidos grasos omega-6",
```

Capitalization looks inconsistent here.
```suggestion
    "omega-6-fat": "Acidos Grasos Omega-6",
```